introduction.md

```diff
@@ -1,4 +1,8 @@
 Hello, my name is Bogdan P
 Hi hello
 test bla bla
-iOS 
\ No newline at end of file
+iOS 
+* Git
+* JavaScript
+* Ruby
+* bla
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?